collector.ts

```diff
@@ -0,0 +1 @@
+import { EventCollector } from 'services/collectors/EventCollector';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Seagate/alex-test/pullRequest?prid=8626240)
```suggestion
import { EventCollector } from "services/collectors/EventCollector";
```